.drone.env

```diff
@@ -5,3 +5,6 @@ CORE_BRANCH=master
 # The test runner source for UI tests
 WEB_COMMITID=0bb33e791f10f1afebfde4ea4da356e46ecdc201
 WEB_BRANCH=master
+
+# e2e tests use the latest stable version on web
+WEB_ASSETS_VERSION=v5.7.0-rc.5
```

Comment:
shouldn't that be the same commit at `WEB_COMMITID`? why do we need that?

Reply:
it is used here to download the web version -> https://github.com/owncloud/ocis/blob/26f15c2fa9abd1767eb0f7d60200404a613a7dc3/services/web/Makefile#L36

Reply:
WEB_COMMITID is only used at `settingsUITests ` & `yarnInstallUITests`

Reply:
```
"git clone -b $WEB_ASSETS_VERSION --single-branch --no-tags https://github.com/owncloud/web.git folder"
```
is same 
```
"git clone -b $WEB_BRANCH --single-branch --no-tags https://github.com/owncloud/web.git folder"
"git checkout $WEB_COMMITID",
```

if WEB_COMMITID matches with WEB_ASSETS_VERSION of web.  I tried to run it again in CI twice and it looks like it works